### PR TITLE
Update the feature launch process doc

### PR DESCRIPTION
**Pull request by syg**

### CODE REVIEW

**Comment by syg** [comment]
@rmahdav PTAL as well, in case I missed something you think we should call out. Can't tag you as a reviewer for some reason...

File: src/docs/feature-launch-process.md [hunk 1]
```diff
@@ -2,21 +2,41 @@
...
 
+### Both V8 and blink flags are required { #flags }
+
+When implementing a feature, both a V8 flag and a blink `base::Feature` are required.
+
+Blink features are required so that Chrome can turn off features without distributing new binaries in emergency situations. This is usually implemented in [`gin/gin_features.h`](https://source.chromium.org/chromium/chromium/src/+/main:gin/gin_features.h), [`gin/gin_features.cc`](https://source.chromium.org/chromium/chromium/src/+/main:gin/gin_features.cc), and [`gin/v8_initializer.cc`](https://source.chromium.org/chromium/chromium/src/+/main:gin/v8_initializer.cc),
+
+### Fuzzing is required to ship { #fuzzing }
+
+JavaScript and WebAssembly features must be fuzzed, and all fuzz bugs must be fixed, before they can be shipped.
```
Comment (dtig) [inline src/docs/feature-launch-process.md:29]: Wdyt of being explicit about a feature being fuzzed for a minimum length of time for fuzzer coverage (4 weeks/1 milestone)?

**Review by dtig — APPROVED** [review]

File: src/docs/feature-launch-process.md (hunk 1 above)
Comment (syg) [reply src/docs/feature-launch-process.md:29]: Yeah, sounds good to me.

**Comment by rmahdav** [comment]
Thanks Shu for updating the document. I think adding the link to the document we created for requesting review gates [go/js-chromestatus](http://goto.google.com/js-chromestatus) somewhere in this doc may also be helpful for users.

**Comment by syg** [comment]
@rmahdav Good suggestion. I wrote a new section, adopted from your doc. PTAL.

@dtig Mind taking a look at the new section as well about the gates wrt Wasm?

**Comment by rmahdav** [comment]
The new section looks good to me. Thanks Shu!

File: src/docs/feature-launch-process.md [hunk 2]
```diff
@@ -2,24 +2,76 @@
...
+#### Enterprise
+
+Throughout their standardization process in TC39 and the Wasm CG, JavaScript and WebAssembly features already undergo heavy backwards compatibility scrutiny. It is exceedingly rare for features to be willfuly backwards incompatible.
+
+For JavaScript, recently shipped features can also be disabled via `chrome://flags/#disable-javascript-harmony-shipping`.
+
+#### Debuggability
+
+JavaScript and WebAssembly features' debuggability differs significantly from feature to feature. JavaScript features that only add new built-in methods do not need additional debugger support, while WebAssembly features that add new capabilities may need significant additional debugger support.
+
```
Comment (dtig) [inline src/docs/feature-launch-process.md:60]: I don't feel strongly about this as this is errata documentation, but would it be useful to surface what debuggability means for language features in more detail? The [Wasm feature debugging checklist](https://goo.gle/devtools-wasm-checklist), is also linked from the shipping checklist for Wasm features, but I don't see a link to the [JS language feature debugging](https://docs.google.com/document/d/1_DBgJ9eowJJwZYtY6HdiyrizzWzwXVkG5Kt8s3TccYE/edit#heading=h.u5lyedo73aa9) checklist linked anywhere else.

**Review by dtig — APPROVED** [review]
Still lgtm.

File: src/docs/feature-launch-process.md (hunk 2 above)
Comment (syg) [reply src/docs/feature-launch-process.md:60]: Great call, will add links.

**Pull request merged**